plugins/screensaverpause/PLUGININFO

```diff
@@ -1,6 +1,6 @@
 Version='4.0.2'
 Authors=['Johannes Sasongko <sasongko@gmail.com>']
 Name=_('Pause on Screensaver')
-Description=_('Pauses (and optionally resumes) playback based on screensaver status.\n\nRequires: GNOME, MATE, Cinnamon, or KDE Screensaver (does not support XScreenSaver nor XLockMore)')
+Description=_('Pauses (and optionally resumes) playback based on screensaver status.\n\nRequires: GNOME, MATE, Cinnamon, XFCE or KDE Screensaver (does not support XScreenSaver nor XLockMore)')
```

Comment:
Since we originally had an Oxford comma here (i.e., after "Cinammon" and before "or"), can you please add it after the newly-added XFCE as well?

Reply:
Added the oxford comma to the description as requested.